src/Features/Core/Portable/LanguageServices/SymbolDisplayService/AbstractSymbolDisplayService.AbstractSymbolDescriptionBuilder.cs

```diff
@@ -687,6 +689,17 @@ private IEnumerable<SymbolDisplayPart> TypeParameterName(string text)
             {
                 return Part(SymbolDisplayPartKind.TypeParameterName, text);
             }
+
+            protected static async Task<List<SymbolDisplayPart>> GetClassifiedSymbolDisplayPartsAsync(
+                SemanticModel semanticModel, TextSpan textSpan, Workspace workspace,
```

Comment:
Would prefer that 'Workspace' was first so that hte parameters go from 'wider' to 'narrower'.